jacoco_report/action_inputs.py

```diff
@@ -521,9 +546,11 @@ def validate_inputs() -> None:
         if not isinstance(fail_symbol, str) or not fail_symbol.strip() or len(fail_symbol) < 1:
             errors.append("'fail-symbol' must be a non-empty string and have a length from 1.")
 
-        fail_on_threshold = ActionInputs.get_fail_on_threshold()
-        if not isinstance(fail_on_threshold, bool):
-            errors.append("'fail-on-threshold' must be a boolean.")
+        fail_on_threshold = None
```

Comment:
This assign met seem s pointless.  The value is not used anywhere and then reassigned two lines below.

Reply:
I cannot find mentioned code in current version.

Reply:
What do you mean ,you cannot find the code?
It's part of the PR freshest version: `fail_on_threshold = None`
Maybe some not pushed commit?